Utils/Dataflow/data4es/batch_stage/run.sh

```diff
@@ -0,0 +1,15 @@
+source ../../shell_lib/eop_filter
+
+cmd="./stage.py -m s"
+cmd_batch2="./stage.py -b 2 -m s"
+cmd_batch100="./stage.py -b 100 -m s"
```

Comment:
Done. You've already suggested this before, and I tried to do it before making this PR, but it was not working for some reason. Did it again after seeing this comment, and it worked. Weird, I probably made a mistake somewhere during the first attempt...

Side note - simply adding `-E ''` works in command line, but not here: due to double quotes, EOP will be set to `''` instead of an empty line, so a workaround has to be used. And no, this is not the reason of the previously mentioned failure, I did it then as well.